scripts/apps/search/MultiImageEdit.ts

```diff
@@ -120,7 +120,19 @@ export function MultiImageEditController(
 
         $scope.images.forEach((item) => {
             if (item.selected) {
-                item[field] = $scope.metadata[field] || '';
+                if (field === 'extra') {
+                    Object.keys(item[field]).forEach((key) => {
+                        if ($scope.metadata[field][key]) {
```

Comment:
If we are writing this code to handle empty strings, could you make the condition `if (typeof x === 'string' && x.trim().length < 1)` so the purpose of the code is clear in the future? Trimming would also be useful in case the string only has whitespace.